src/DataProvider/DenormalizedIdentifiersAwareItemDataProviderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\DataProvider;
+
+/**
+ * Retrieves items from a persistence layer.
```

Comment:
Can you change this sentence by something like `Marks data providers able to deal with complex identifiers denormalized as an array`?